bin/mksetup

```diff
@@ -7,7 +7,7 @@ try:
     eupsdir = sys.argv.pop(0)
     prefix = sys.argv.pop(0)
 except IndexError:
-    print >> sys.stderr, "Usage: mksetup EUPS_DIR EUPS_PATH [SETUP_ALIASES]"
+    sys.stderr.write("Usage: mksetup EUPS_DIR EUPS_PATH [SETUP_ALIASES]\n")
```

Comment:
@timj I think that makes sense. Both ways work so it is a matter of personal preference.